content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md

```diff
@@ -17,6 +17,36 @@ weight: 1
 
 # Corda Enterprise release notes
 
+## Corda Enterprise 4.8.8
+
+Corda Enterprise 4.8.8 is a patch release of Corda Enterprise focused on CorDapp response time and security improvements.
+
+### Upgrade recommendation
+
+As a developer, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and you can find the latest upgrade guide [here](../../../../../en/platform/corda/4.9/enterprise/upgrading-index.md).
+
+As a node operator, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html).
+
+### Fixed issues
+
+In this patch release:
+* Java serialization has been disabled in the Corda firewall, closing a potential security vulnerability.
+* The time it takes for first-time flows to be run in a signed CorDapp has been significantly reduced. Backporting `setUseCaches(false)` from Quasar 0.8 to Quasar 0.7.15_r3 had measurably increased the node’s start-up time. Removing `setUseCaches(false)` restored this lost performance and reduced node start-up time.
```

Comment:
The slowness was not caused by the back-porting. See previous comment.

This fixes a regression introduced in Corda version...? (By [CORDA-4120](https://r3-cev.atlassian.net/browse/CORDA-4120)) (Blah blah..)